### PR TITLE
Describe details about incubating status

### PR DESCRIPTION
**Pull request by tisonkun**
According to the vote and related discussion - https://lists.apache.org/thread/1kf8spl8z8m91773yqzxbhx27mrhxhdg

### CODE REVIEW

File: pages/guides/branding.ad [hunk 1]
```diff
@@ -41,7 +41,7 @@ _Podling-Name_' (see link:#Naming[Naming] below).
 A podling and affiliated persons can issue press releases that
 reference the podling, but cannot issue press releases with the
 specific intent of announcing the podling. The podling can conduct
-informal PR activities such as media outreach, blog publicity, etc.
+informal press releases activities such as media outreach, blog publicity, etc.
```
Comment (tisonkun) [inline pages/guides/branding.ad:44]: PR is ambiguous as "pull request" and we use always "press releases" elsewhere in this page.

File: pages/guides/branding.ad [hunk 2]
```diff
@@ -60,13 +60,15 @@ joint press release with any other affiliated organizations or groups.
 After a podling has been approved, the lists are created, and the initial
  code drop has commenced, you *MUST* refer to the podling as `Apache
  _Podling-Name_` *AND* mention that the project is
- under Incubation.  Suitable mentions include:
+ under Incubation. Suitable mentions include:
 
-- Inclusion of the `http://_podling-name_.incubator.apache.org/` URL
```
Comment (tisonkun) [inline pages/guides/branding.ad:65]: We deprecated `incubator` in URL.

File: pages/guides/branding.ad [hunk 3]
```diff
@@ -60,13 +60,15 @@ joint press release with any other affiliated organizations or groups.
...
 - `Apache _Podling-Name_ is currently undergoing Incubation at the Apache Software Foundation`.
+- Referring the podling as `Apache _Podling-Name_ (Incubating)`.
 
 The Incubator PMC (IPMC) must approve other references before you publish them. You only need to make these statements upon the first reference to the podling in a document.
 
+Specifically, you SHOULD include these statements in the README file of each repository belonging to the podling. You CAN add a link to the link:#Disclaimers[Disclaimers] in the README file. You CAN also refer the podling as `Apache _Podling-Name_ (Incubating)` in the GitHub repository description.
```
Comment (tisonkun) [inline pages/guides/branding.ad:70]: Hopefully this describes our guidelines correctly and clearly.

File: pages/guides/publicity.ad [hunk 4]
```diff
@@ -50,10 +50,9 @@ As all Apache PMCs (and Incubator PPMCs) are responsible for the "care and feedi
 
 - Ensure your https://projects.apache.org/[project listing and DOAP (Description Of A Project) file] accurately reflects what the podling is about. Be succinct, particularly with the short description (note that the project may fall under one category): this helps those unfamiliar with your project find you. Use the longer project description as the official project boilerplate ("About Apache _Project Name_") and, for consistency, include it in official news announcements, media briefings, and other publications (the ASF Boilerplate is for Foundation-originating press releases and is **not** to be used in any project announcements).
 - Upload a high-resolution version of your project’s logo at http://www.apache.org/logos/[apache.org/logos/].
-- Contact ASF Infrastructure to establish a https://blogs.apache.org/_PROJECTNAME_ project blog directory (and admin/publishing credentials).
```
Comment (tisonkun) [inline pages/guides/publicity.ad:53]: blogs.apache.org has been archived.

File: pages/guides/publicity.ad [hunk 5]
```diff
@@ -50,10 +50,9 @@ As all Apache PMCs (and Incubator PPMCs) are responsible for the "care and feedi
...
 - Request a project interview on https://feathercast.apache.org/[Feathercast, the voice of the ASF].
 - Gain additional exposure by posting newsworthy podling updates and milestones to http://apache.org/foundation/mailinglists.html#foundation-announce[announce(at)apache(dot)org]. The Apache team adds these to the Apache Weekly News Round-Ups, which it also sends to members of the media and analyst community.
-- Establish a Twitter (and/or other social media) account(s) with a handle that includes "Apache" or "ASF" with the project name and description (example at https://twitter.com/ApacheAccumulo[https://twitter.com/ApacheAccumulo]). Share information often, and point to your podling Website and email list archives where possible. 
+- Establish a X (formerly Twitter) (and/or other social media) account(s) with a handle that includes "Apache" or "ASF" with the project name and description (example at https://x.com/ApacheArrow[https://x.com/ApacheArrow]). Share information often, and point to your podling Website and email list archives where possible.
```
Comment (tisonkun) [inline pages/guides/publicity.ad:55]: ApacheAccumulo account has been archived.

**Review by jpmaniqis — APPROVED** [review]

File: pages/guides/branding.ad (hunk 1 above)
Comment (dave2wave) [reply pages/guides/branding.ad:44]: c/press releases/press release/

File: pages/guides/branding.ad (hunk 3 above)
Comment (dave2wave) [reply pages/guides/branding.ad:70]: s/SHOULD/MUST/ - the disclaimer must be present - it is required. We should be gentle when we find it is missing but really need to insist.

s/CAN/SHOULD/ - CAN is not prescriptive. SHOULD is a strong suggestion.

File: pages/guides/publicity.ad [hunk 6]
```diff
@@ -40,7 +40,7 @@ You may not disseminate formal press releases or announcements of any kind on an
...
+When speaking with members of the media and analyst community, you must refer to the podling as "Apache _Podling-Name_" as per the Branding Guidelines, along with explaining the requisite podling disclaimer(s). Be prepared to answer "Who uses you?" As a podling’s known user base grows, PMCs (Project Management Committees) are welcome to promote their users on a "Powered By" page such as https://cwiki.apache.org/hadoop/PoweredBy[Apache Hadoop's Powered By page]. Create and promote podling users to https://incubator.apache.org/guides/press-kit.html[display the "Powered By" logo]. Also, ensure that the podling maintains http://apache.org/press/media.html#interviews[*vendor neutrality*], particularly when referring to project users.
```
Comment (dave2wave) [inline pages/guides/publicity.ad:43]: s|http://apache.org|https://apache.org|

File: pages/guides/publicity.ad (hunk 5 above)
Comment (dave2wave) [reply pages/guides/publicity.ad:55]: You should say something about the PPMC sharing credentials and making decisions about the X account's post. If this is not established then years later a PMC may have some issues taking control of their social media.

File: pages/guides/branding.ad (hunk 1 above)
Comment (tisonkun) [reply pages/guides/branding.ad:44]: Good catch. Fixed.

File: pages/guides/branding.ad (hunk 3 above)
Comment (tisonkun) [reply pages/guides/branding.ad:70]: Make sense. Updated.

File: pages/guides/publicity.ad (hunk 6 above)
Comment (tisonkun) [reply pages/guides/publicity.ad:43]: Updated.

File: pages/guides/branding.ad [hunk 7]
```diff
@@ -60,13 +60,15 @@ joint press release with any other affiliated organizations or groups.
 After a podling has been approved, the lists are created, and the initial
  code drop has commenced, you *MUST* refer to the podling as `Apache
  _Podling-Name_` *AND* mention that the project is
- under Incubation.  Suitable mentions include:
+ under Incubation. Suitable mentions include:
 
-- Inclusion of the `http://_podling-name_.incubator.apache.org/` URL
 - `Apache _Podling-Name_ is currently undergoing Incubation at the Apache Software Foundation`.
+- Referring the podling as `Apache _Podling-Name_ (Incubating)`.
```
[the remark this answers is not in the document]
Comment (tisonkun) [reply pages/guides/branding.ad:66]: Updated.

File: pages/guides/publicity.ad (hunk 5 above)
[the remark this answers is not in the document]
Comment (tisonkun) [reply pages/guides/publicity.ad:55]: Updated.

Comment (tisonkun) [reply pages/guides/publicity.ad]: Added in a7419ddce2dca949ecbd4840eee07f3ab05dd278.

File: pages/guides/publicity.ad (hunk 6 above)
[the remark this answers is not in the document]
Comment (tisonkun) [reply pages/guides/publicity.ad]: Rephrase in a7419ddce2dca949ecbd4840eee07f3ab05dd278.

**Comment by tisonkun** [comment]
@sebbASF @dave2wave Thanks for your reviews. Comments addressed.

**Comment by tisonkun** [comment]
I'm going to merge this patch in this week. Would you take another round here or shall I wait for your review @sebbASF @dave2wave @justinmclean?

**Review by dave2wave — APPROVED** [review]

**Comment by tisonkun** [comment]
Merged. Welcome to leave reviews after merge.

**Pull request merged**